### ISSUE TITLE
fix(dgeni): correctly export type declarations

### ISSUE THREAD

**Issue by jeffbcross** [angular/angular#3828]
For `type` declarations, such as [this one](https://github.com/angular/angular/commit/512340e39b7858d1bf7c7751714cbf69220aac84#diff-a20f042e0a0b324d9daec6988a9774d8R45) in http interfaces, the d.ts output is `type-alias` instead of `type`. This was broken for http.d.ts in master, but apparently was not being tested. Tests now fail with "semicolon expected."

Input:
```typescript
export type RequestOptionsArgs = {
  url?: string;
  method?: RequestMethods;
  search?: string | URLSearchParams;
  headers?: Headers;
  // TODO: Support Blob, ArrayBuffer, JSON, URLSearchParams, FormData
  body?: string;
  mode?: RequestModesOpts;
  credentials?: RequestCredentialsOpts;
  cache?: RequestCacheOpts;
}
```

Output:
```typescript
type-alias RequestOptionsArgs;
```

**Comment by jeffbcross** [comment]
CC: @alexeagle @mhevery

**Comment by mhevery** [comment]
@jeffbcross pleases don't add more then one component to the issue. It confuses the dashboard. If you have a use case for it lets chat.

Dup of https://github.com/angular/angular/issues/3699

**Comment by jeffbcross** [comment]
@mhevery good to know about the component. This doesn't seem like a dupe of #3699, as this is pointing out a bug when using `type` assignments.

**Comment by jeffbcross** [comment]
Closed by @IgorMinar with d78261695b3db304501063e69f600252d5e6f445
